yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -28,6 +28,30 @@ methods:
           - xyz.openbmc_project.Common.Error.InvalidArgument
           - xyz.openbmc_project.Common.Error.InternalFailure
 
+    - name: WriteKeywordOnHardware
+      description: >
+          Method to update a single keyword's value on the hardware.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                EEPROM path of the FRU.
+          - name: paramsToWriteData
```

Comment:
We will need offset variable as well. Default it to zero. Add detail in description for the same.